app/views/corporate/extra/_our_debt_work_corporate.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="l-inpage-nav">
+  <div class="l-inpage-nav__item">
+    <ul class="link-list-primary link-list-primary--no-border">
+      <li class="link-list-primary__item">
+        <a href="#" class="link-list-primary__text is-selected">Lorem ipsum</a>
```

Comment:
I don't actually know what these partials are used for. @ourmaninamsterdam any idea?